lib/cssom.js

```diff
@@ -58,5 +333,4 @@ declare class CSSStyleDeclaration {
   removeProperty(property: string): string;
   setProperty(property: string, value: ?string, priority: ?string): void;
   setPropertyPriority(property: string, priority: string): void;
-  setPropertyValue(property: string, value: string): void;
```

Comment:
I don't have a strong opinion either. I tried to use that function and it didn't work on any browser, checked the standard and it was not there, so I assumed it was a copy & paste error.

Let me know if you want it back and I will update the PR.